grc/compiler.py

```diff
@@ -42,12 +42,8 @@ def main(args=None):
     )
     platform.build_library()
 
-    out_dir = args.output if not args.user_lib_dir else platform.config.hier_block_lib_dir
-    if os.path.exists(out_dir):
-        pass  # all is well
-    elif args.save_to_lib:
-        os.mkdir(out_dir)  # create missing hier_block lib directory
-    else:
+    output_dir = args.output if not args.user_lib_dir else platform.config.hier_block_lib_dir
+    if not os.path.exists(output_dir):
         exit('Error: Invalid output directory')
```

Comment:
Why do you stop if the directory does not exist instead of creating the directory

Reply:
I don't think the old code did either. That args.save_to_lib was never true, there's no CLI switch for it. Perhaps in a different call to this but I couldn't find a reference to save_to_lib anywhere else in the code. So I simply pruned that seemingly dead branch.

I tend to prefer explicit user action vs. a "side-effect", but perhaps specifying a non-existent directory on the CL is explicit enough.